stories/SQForm.stories.tsx

```diff
@@ -852,6 +853,147 @@ export const ccaChecklist = (): JSX.Element => {
   );
 };
 
+export const paymentForm = (): JSX.Element => {
+  const validationSchema = Yup.object({
+    billingAddress1: Yup.string().required(),
+    billingAddress2: Yup.string(),
+    city: Yup.string().required(),
+    state: Yup.string().required(),
+    zip: Yup.string().required(),
+    cardholderName: Yup.string().required(),
+    cardNumber: Yup.string().required(),
+    expiry: Yup.string().required(),
+    cvv: Yup.string().required(),
+  });
+
+  const initialValues = {
+    billingAddress1: '',
+    billingAddress2: '',
+    city: '',
+    state: '',
+    zip: '',
+    cardholderName: '',
+    cardNumber: '',
+    expiry: '',
+    cvv: '',
+  };
+
+  return (
+    <Card raised style={{padding: 16, width: '768px'}}>
+      <SQForm
+        initialValues={initialValues}
+        onSubmit={handleSubmit}
+        validationSchema={validationSchema}
+      >
+        <SQFormTextarea
+          label="Billing Address 1 *"
```

Comment:
Whats up with these asterisks? If you _need_ these you can add them back with css. The theme removes them so you can add them back in your project as you need.